site/src/components/radio.md

```diff
@@ -0,0 +1,159 @@
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04240"
+                    		value="4"
+                            class="radio-dot"
+                    	/>
+                    	<span class="radio-label-text">50%</span>
+                    </label>
+				</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		id="assessment-choice-b16fe0d7-9dfa-460d-a8c4-8b39cad04240-4"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04240"
+                    		value="4"
+                            class="radio-dot"
+                            disabled
+                    	/>
+                    	<span class="radio-label-text">70%</span>
+                    </label>
+				</td>
+			</tr>
+			<tr>
+				<td>Checked</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		id="assessment-choice-b16fe0d7-9dfa-460d-a8c4-8b39cad04240-4"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04241"
+                    		value="4"
+                            class="radio-dot"
+                            checked
+                    	/>
+                    	<span class="radio-label-text">50%</span>
+                    </label>
+				</td>
+				<td>
+					<label class="radio" title="">
+                    	<input
+                    		type="radio"
+                    		id="assessment-choice-b16fe0d7-9dfa-460d-a8c4-8b39cad04240-4"
+                    		name="b16fe0d7-9dfa-460d-a8c4-8b39cad04241"
+                    		value="4"
+                    		class="is-checked radio-dot"
+                            disabled
+                    	/>
+                    	<span class="radio-label-text">70%</span>
+                    </label>
+				</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+## Usage
+
+Here is an example of a group of radio buttons. You can group multiple ratio buttons together within a `fieldset`.
+Make sure to use the same value for the `name` attribute within a group of radio buttons.
+
+```html
+<fieldset class="field border-none">
+	<legend>This legend describes the nature of the selection.</legend>
+	<label class="radio">
+		<input checked name="question-1" type="radio" class="radio-dot" />
+		<span class="radio-label-text">Yes</span>
+	</label>
+	<label class="radio">
+		<input name="question-1" type="radio" class="radio-dot" />
+		<span class="radio-label-text">No</span>
+	</label>
+	<label class="radio">
+		<input name="question-1" type="radio" class="radio-dot" />
+		<span class="radio-label-text">Maybe</span>
+	</label>
+</fieldset>
+```
+
+## Modifiers
+
+### Styles
+
+`is-checked` can be used with `.radio-dot` to create the appearance of a radio button with a checked state, without actually setting the state.
+
+```html
+<fieldset class="field border-none">
```

Comment:
Not sure the text in this example is correctly aligned with the dot:
![image](https://user-images.githubusercontent.com/30843002/157938024-ca42b30c-cb06-43ac-93a3-dd49c251d638.png)

Reply:
Added `inset-block-start` to `radio-dot` to lower the radio down a bit, without making it look odd for single line text.